### DISCUSSION TITLE
Interaction counts - change or settings?

### DISCUSSION THREAD

**Discussion by IBBoard** [CodedOre/NewCaw#62] [Design]
One thing that I like with Mastodon web and the Megalodon client on Android is the lack of emphasis on interaction counts.

Mastodon Web just shows "0", "1" or "1+" for reply count and no boost/like count in the main timeline. Megalodon doesn't show any counts in timelines. Both still show counts in the individual toot view.

I think there's two ways we can approach this:
1. Be opinionated and take one of those two approaches.
2. Give the user the options of: All, reply count summary, or none

I think we should implement it for Twitter as well, even though it's different to their web design (but Musk has added view counts, and I've already seen people saying that it's good that Cawbird doesn't show that junk!)

**Comment by CodedOre** [comment]
That's an interesting idea. I assume the idea is to move the focus from the interaction count toward the content of the post, right?

About the implementation, I would say that we don't need an option for this. The more interesting question is on how we approach the design.\
We still need the buttons to allow to eventually like, repost, etc. a post also in the timeline. But we could move the numbers to a separate widget, like dicussed [here](https://github.com/CodedOre/NewCaw/discussions/34#discussioncomment-2393642).

**Reply by IBBoard** [reply]
Yeah, I've read something about it being about emphasising the content rather than making people chase interaction counts (or be drawn to the content in the timeline that already has a high interaction count).

The buttons remain. It's just about whether we put numbers next to them or not in the timelines.

Example from Megalodon:
![image](https://user-images.githubusercontent.com/388582/210179532-33d6858b-b45c-4407-9376-9ed673604782.png)
(Also highlighted: How they make "boosted" vs "not boosted" more accessible)
